holidays/calendars/sinhala_astronomy.py

```
@@ -0,0 +1,221 @@
+    start = ts.utc(year, 1, 1)
+    end = ts.utc(year + 1, 1, 1)
+
+    times, phases = almanac.find_discrete(
+        start,
+        end,
+        almanac.moon_phases(eph),
+    )
+
+    full_moons = []
+
+    for t, phase in zip(times, phases):
```

Ruff (0.15.20)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)